packages/front-end/components/InitialSetup/SelectDataSourcePage.tsx

```diff
@@ -0,0 +1,155 @@
+import { PiInfo, PiPaperPlaneTiltFill } from "react-icons/pi";
+import { useState } from "react";
+import { SchemaFormat } from "back-end/types/datasource";
+import { Callout } from "@radix-ui/themes";
+import clsx from "clsx";
+import { eventSchemas } from "@/services/eventSchema";
+import NewDataSourceForm from "@/components/Settings/NewDataSourceForm";
+import DataSourceLogo from "@/components/DataSources/DataSourceLogo";
+import { useDefinitions } from "@/services/DefinitionsContext";
+import { useDemoDataSourceProject } from "@/hooks/useDemoDataSourceProject";
+import InviteModal from "@/components/Settings/Team/InviteModal";
+import { useUser } from "@/services/UserContext";
+import { useLocalStorage } from "@/hooks/useLocalStorage";
+import styles from "./InitialSetup.module.scss";
+
+interface Props {
+  onSuccess: () => void;
+}
+
+const SelectDataSourcePage = ({ onSuccess }: Props) => {
+  const [newModalOpen, setNewModalOpen] = useState(false);
+  const [inviting, setInviting] = useState(false);
+  const [
+    setupEventTracker,
+    setSetupEventTracker,
+  ] = useLocalStorage<SchemaFormat | null>(`setup_event_tracker`, null);
+  const [eventTracker, setEventTracker] = useState<null | SchemaFormat>(
+    setupEventTracker
+  );
+
+  const { refreshOrganization } = useUser();
+  const { mutateDefinitions } = useDefinitions();
+  const { exists: demoDataSourceExists } = useDemoDataSourceProject();
+
+  return (
+    <>
+      {inviting && (
+        <InviteModal
+          close={() => setInviting(false)}
+          mutate={refreshOrganization}
+        />
+      )}
+      {newModalOpen && eventTracker && (
+        <NewDataSourceForm
+          existing={false}
+          data={{
+            settings: {
+              schemaFormat: eventTracker,
+            },
+          }}
+          source="setup-flow"
+          onSuccess={async () => {
+            await mutateDefinitions({});
+            setNewModalOpen(false);
+            onSuccess();
+          }}
+          onCancel={() => {
+            setNewModalOpen(false);
+            setSetupEventTracker(eventTracker);
```

Comment:
Hmm maybe this actually needs to be stored by organization for multi-org setups. I can also just store this on the organization if we want anyone in the org with the right permission to be able to finish the datasource setup